grakn-graql/src/main/java/ai/grakn/graql/internal/util/StringConverter.java

```diff
@@ -48,48 +46,6 @@
     private StringConverter() {}
 
     /**
-     * @param string the string to unescape
-     * @return the unescaped string, replacing any backslash escapes with the real characters
-     */
-    public static String unescapeString(String string) {
-        return StringEscapeUtils.unescapeJavaScript(string);
-    }
-
-    /**
-     * @param string the string to escape
-     * @return the escaped string, replacing any escapable characters with backslashes
-     */
-    public static String escapeString(String string) {
-        return StringEscapeUtils.escapeJavaScript(string);
-    }
-
-    /**
-     * @param string a string to quote and escape
-     * @return a string, surrounded with double quotes and escaped
-     */
-    private static String quoteString(String string) {
-        return "\"" + escapeString(string) + "\"";
-    }
-
-    /**
-     * @param value a value in the graph
-     * @return the string representation of the value (using quotes if it is already a string)
-     */
-    public static String valueToString(Object value) {
-        if (value instanceof String) {
-            return quoteString((String) value);
-        } else if (value instanceof Double) {
-            DecimalFormat df = new DecimalFormat("#", DecimalFormatSymbols.getInstance(Locale.ENGLISH));
-            df.setMinimumFractionDigits(1);
-            df.setMaximumFractionDigits(12);
-            df.setMinimumIntegerDigits(1);
-            return df.format(value);
-        } else {
-            return value.toString();
-        }
-    }
-
-    /**
```

Comment:
Ideally we should see if all of this can be moved to `StringUtil`, instead of having two `String` helper classes.